docs/src/content/docs/it/reference/frontmatter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Riferimenti frontmatter 
+description: Una panoramica sui campi predefiniti del frontmatter Starlight.
+---
+
+Puoi personalizzare pagine Markdown e MDX in Starlight definendo i valori nel frontmatter. Per esempio, una pagina potrebbe definire `title` e `description` :
+
+```md
+---
+title: A proposito del progetto
+description: Scopri di più sul progetto a cui sto lavorando.
+---
+
+Benvenuto alla pagina "a proposito del progetto"!
+```
+
+## Campi del frontmatter
+
+### `title` (obbligatorio)
+
+**type:** `string`
+
+Devi fornire un titolo ad ogni pagina. Questo sarà usato in testa alla pagina, nelle finestre del browser e nei metadati della pagina.
+
+### `description`
+
+**type:** `string`
+
+La descrizione è utilizzata nei metadati e sarà utilizzata dai motori di ricerca e nelle anteprime nei social.
+
+### `editUrl`
+
+**type:** `string | boolean`
+
+Sovrascrive la [configurazione globale `editLink`](/it/reference/configuration/#editlink). Metti a `false` per disabilitare "Modifica la pagina" per quella specifica oppure fornisci un link alternativo.
```

Comment:
Qua manca “pagina” credo?

```suggestion
Sovrascrive la [configurazione globale `editLink`](/it/reference/configuration/#editlink). Metti a `false` per disabilitare "Modifica la pagina" per quella pagina specifica oppure fornisci un link alternativo.
```

Reply:
In questo punto l'idea era di non ripetere "pagina", siccome già presente in "Modifica la pagina". Tuttavia, nonostante la ripetizione, in questo caso effettivamente è più comprensibile come da suggerimento